libparsec/crates/types/src/local_device_file.rs

```diff
@@ -153,15 +153,14 @@ impl DeviceFileType {
     }
 }
 
-#[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize)]
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
```

Comment:
What is the reason for the `Serialize` to be remove, we no longer want `AvailableDevice` to be serializable ?

Reply:
`AvailableDevice` is never serialized (it is only provided to the GUI to list the devices detected on disk)

Reply:
thanks